src/main/java/burgerburglar/Ui.java

```diff
@@ -0,0 +1,95 @@
+package burgerburglar;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.Scanner;
+
+/**
+ * Handles all user interactions for BurgerBurglar.
+ * <p>
+ * Responsibilities include reading user input, printing messages, errors,
+ * welcome/goodbye messages, task lists, and the user manual.
+ */
+public class Ui {
+    private static final String LINE_BREAK =
+            "______________________________________________________________________";
+
+    private final Scanner scanner = new Scanner(System.in);
+
+    /**
+     * Shows the welcome message, including the BurgerBurglar logo and version,
+     * followed by a user manual.
+     *
+     * @param logo    the ASCII art logo
+     * @param version the version string
+     */
+    public void showWelcome(String logo, String version) {
+        System.out.println(logo + "INITIATING BURGERBURGLAR " + version + "\n"
+                + "------------------------------------------------------------------100%\n");
+        showLine();
+        printUserManual();
+        showLine();
+        System.out.println("GOOD DAY, GOOD BURGER.\nWHAT CAN BURGER STEAL FOR YOU?");
+    }
+
+    /**
+     * Shows the goodbye message.
+     *
+     * @param version the version string
+     */
+    public void showGoodbye(String version) {
+        showLine();
+        System.out.println("GOODBYE, GOODBURGER.");
+        showLine();
+        System.out.println("EXITING BURGERBURGLAR " + version);
+    }
+
+    // Print a line break
+    public void showLine() {
+        System.out.println(LINE_BREAK);
+    }
+
+    // Read input from the user
```

Comment:
Should this be in JavaDoc format?
```suggestion
    /** Reads input from the user */
```